reports/tests.py

```diff
@@ -528,3 +533,159 @@ def _generate_invoice_date(year=datetime.now().year, month=1, this_month=False):
             date = datetime(year, month, 1)
 
         return timezone.make_aware(date)
+
+
+class ReportTaskTestCase(TestCase):
+    """
+    Tests for reports tasks execution
+    """
+
+    def create_user_with_trial_instance(self, key=''):
+        """
+        Returns a trial instance for a user
+
+        :param key: String to append to properties.
+        :return: A trial instance
+        """
+        user, _ = get_user_model().objects.get_or_create(
+            username=key + 'test', email=key + 'test@example.com')
+        instance = OpenEdXInstanceFactory(
+            successfully_provisioned=True, betatestapplication=BetaTestApplication())
```

Comment:
@spokerman12, nit: formatting of closing parenthesis doesn't match the coding standards that we follow.

Reply:
Fixed plus another occurrence.